gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import CodeExample from '../../CodeExample';
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
+import { NxP, NxCode, NxTable, NxTile, NxH2, NxH3 } from '@sonatype/react-shared-components';
+
+import './NxGlobalSidebarPage.scss';
+
+const NxGlobalSidebarExample = require('./NxGlobalSidebarExample.tsx?raw');
+
+export default function NxGlobalSidebarPage() {
+  return (
+    <>
+      <GalleryDescriptionTile>
+        <NxP>
+          <NxCode>NxGlobalSidebar</NxCode> is a collapsible page level sidebar. It has two main sections:
+        </NxP>
+        <ul className="nx-list nx-list--bulleted">
+          <li className="nx-list__item">A header which contains branding as well as the open/close toggle</li>
+          <li className="nx-list__item">
+            A navigation link section which display an icon plus text in the open state and an icon in the closed
+            state
+          </li>
+        </ul>
+        <NxP>
```

Comment:
Should mention that it should only be used with section scrolling and full-width mode

Reply:
> Should mention that it should only be used with section scrolling and full-width mode

Better docs here: https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/5f3c9d3f3313a84fe9cb7f904e3d0b504d50393c